components/cam/src/physics/cam/micro_p3.F90

```diff
@@ -2234,12 +2235,12 @@ subroutine ice_cldliq_wet_growth(rho,t,pres,rhofaci,    &
 
    real(rtype) :: qsat0, dum, dum1
 
-   if (qi_incld.ge.qsmall .and. qc_incld+qr_incld.ge.1.e-6_rtype .and. t.lt.zerodegc) then
-      qsat0=qv_sat( zerodegc,pres,0 )
+   if (qi_incld.ge.qsmall .and. qc_incld+qr_incld.ge.1.e-6_rtype .and. t_atm.lt.T_zerodegc) then
+      qsat0=qv_sat( T_zerodegc,pres,0 )
```

Comment:
Also, can we change T_zerodegc to t_zerodegc so there's a consistent rule about capitalizing T?

Reply:
Also for T_rainfrz... maybe search + replace T_?

Reply:
Actually, nevermind. I see that the C++ uses uppercase T consistently for everything. Not worth worrying about the F90 since we will throw it away.